src/components/Content/index.tsx

```diff
@@ -2,9 +2,13 @@ import React from 'react'
 import styled from 'styled-components'
 
 const ContentDiv = styled.div`
+  margin-top: 80px;
   width: 100%;
   overflow-x: hidden;
   flex: 1;
+  @media (max-width: 700px) {
+    margin-top: 42px;
```

Comment:
`margin-top` will be updated when search input bar appears by clicking search icon on mobile.